### ISSUE TITLE
[a11y]: Pagination, PaginationNav - Context lost  when zoomed to 400%.

### ISSUE THREAD

**Issue by lisawu-1** [carbon-design-system/carbon#16080]
### Package

@carbon/react

### Operating System

MacOS

### Package version

1.50.0

### React version

17.0.2

### Automated testing tool and ruleset

https://www.ibm.com/able/toolkit/design/visual/#responsive-reflow

### Description

<Pagination> component
Items per page and page select are missing when zoomed to 400%.
Expected: no content lost
Criteria: https://www.ibm.com/able/toolkit/design/visual/#responsive-reflow

### Reproduction/example

Items per page and page select are missing when zoomed to 400%.

### Steps to reproduce

Zoom to 400%

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Hey there, this is intentional behavior of these components. They're responsive and designed with progressive disclosure in mind. At smaller screens (or high zoom levels) the component omits all but the necessary information/controls.


https://github.com/carbon-design-system/carbon/assets/3360588/f5dca53d-a3f3-4e28-b2b0-2edbe29df40a
